src/licensedcode/data/licenses/drl-1.1.LICENSE

```diff
@@ -5,44 +5,27 @@ name: Detection Rule License (DRL) 1.1
 category: Permissive
 owner: Florian Roth
 homepage_url: https://github.com/Neo23x0/signature-base/blob/master/LICENSE
-spdx_license_key: LicenseRef-scancode-drl-1.1
+spdx_license_key: DRL-1.1
+other_spdx_license_keys:
+    - LicenseRef-scancode-drl-1.1
+other_urls:
+    - https://github.com/SigmaHQ/Detection-Rule-License/blob/6ec7fbde6101d101b5b5d1fcb8f9b69fbc76c04a/LICENSE.Detection.Rules.md
 ---
 
 Detection Rule License (DRL) 1.1
 
-Permission is hereby granted, free of charge, to any person obtaining a copy 
-of this rule set and associated documentation files (the "Rules"), to deal 
-in the Rules without restriction, including without limitation the rights to 
-use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies 
-of the Rules, and to permit persons to whom the Rules are furnished to do so, 
-subject to the following conditions:
-
-If you share the Rules (including in modified form), you must retain the 
-following if it is supplied within the Rules:
-
-1. identification of the authors(s) ("author" field) of the Rule and any 
-   others designated to receive attribution, in any reasonable manner 
-   requested by the Rule author (including by pseudonym if designated).
-
-2. a URI or hyperlink to the Rule set or explicit Rule to the extent 
-   reasonably practicable
-
-3. indicate the Rules are licensed under this Detection Rule License, and 
-   include the text of, or the URI or hyperlink to, this Detection Rule 
-   License to the extent reasonably practicable
-
-If you use the Rules (including in modified form) on data, messages based on 
-matches with the Rules must retain the following if it is supplied within the 
-Rules:
-
-1. identification of the authors(s) ("author" field) of the Rule and any 
-   others designated to receive attribution, in any reasonable manner 
-   requested by the Rule author (including by pseudonym if designated).
-
-THE RULES ARE PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
-OUT OF OR IN CONNECTION WITH THE RULES OR THE USE OR OTHER DEALINGS IN THE 
-RULES.
\ No newline at end of file
+Permission is hereby granted, free of charge, to any person obtaining a copy of this rule set and associated documentation files (the "Rules"), to deal in the Rules without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Rules, and to permit persons to whom the Rules are furnished to do so, subject to the following conditions:
```

Comment:
Why did you replace our text which is better and properly formatted?
You should IMHO keep our text and add the SPDX text as a detection rule instead.